web-main/src/app/components/pages/cons-page/components/cons-header/cons-header.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="conversations-header">
   <div class="conversations-header__logo">logo</div>
   <div class="conversations-header__selector">selector</div>
-  <div class="conversations-header__info">info</div>
+  <div class="conversations-header__info">{{currentlyLoggedUserSg()?.name}}</div>
```

Comment:
I case we don't have `currentlyLoggedUserSg()`, in other words it returns undefined, we should add some indicator, something like _None_ or _Loading_